source/dataman/CacheMan.h

```diff
@@ -7,6 +7,7 @@
  *  Created on: Apr 18, 2017
  *      Author: Jason Wang
  */
+
 #ifndef CACHEMAN_H_
```

Comment:
In order to avoid issues with conflicting include guards, we're starting to use ifdef guards based on the actual path, not just the filename.  Please adjust all the include guards to use `#ifdef DATAMAN_FILENAME_H_`